xml/issue2801.xml

```diff
@@ -26,9 +26,139 @@ satisfied.
 </p>
 <note>Issues Telecom 16-Dec-2016</note>
 <p>Priority 2; Howard and Ville to provide wording.</p>
+
+<note>2016-12-24: Howard and Ville provided wording.</note>
+
 </discussion>
 
 <resolution>
+
+<ol>
+<li>
+<p>
+Modify the synopsis in  <sref ref="[unique.ptr.single]"/> as follows:
+</p>
+<blockquote>
+<pre>
+...
+constexpr unique_ptr(nullptr_t) noexcept<ins>;</ins>
+    <del>: unique_ptr() { }</del>
+...
+</pre>
+</blockquote>
+</li>
+
+<li>
+<p>
+Modify <sref ref="[unique.ptr.single.ctor]"/> as follows:
+</p>
+<blockquote>
+
+<pre>
+constexpr unique_ptr() noexcept;
+</pre>
+<blockquote>
+<p>...</p>
+<p>
+4 <i>Remarks:</i> <del>If this constructor is instantiated with a
+pointer type or reference type for the template argument <tt>D</tt>,
+the program is ill-formed.</del> <ins>If <tt>deleter_type</tt> is a
+pointer type or is not <tt>DefaultConstructible</tt> (Table 22), this
+constructor shall not participate in overload resolution.</ins>
+</p>
+</blockquote>
+
+<pre>
+explicit unique_ptr(pointer p) noexcept;
+</pre>
+<blockquote>
+<p>...</p>
+<p>
+8 <i>Remarks:</i> <del>If this constructor is instantiated with a
+pointer type or reference type for the template argument <tt>D</tt>,
+the program is ill-formed.</del> <ins>If <tt>deleter_type</tt> is a
+pointer type or is not <tt>DefaultConstructible</tt> (Table 22), this
+constructor shall not participate in overload resolution.</ins>
+</p>
+</blockquote>
+
+<pre>
+<ins>constexpr unique_ptr(nullptr_t) noexcept;</ins>
+</pre>
+<blockquote>
+<p>
+<ins><i>Requires:</i> <tt>deleter_type</tt> shall satisfy the requirements of
```

Comment:
I think this paragraph is implied by the "equivalent to" magic words, but doesn't hurt.